conartist.js

```diff
@@ -0,0 +1,8 @@
+const { config, preset } = require('conartist');
```

Comment:
Can remove this if we don't want to use it moving forward. It was just helpful in setting up a complete repo in a matter of seconds.